src/eventsub/mod.rs

```diff
@@ -747,6 +747,57 @@ pub struct EventSubSubscription {
     pub version: String,
 }
 
+pub(crate) trait NamedField {
+    const NAME: &'static str;
+}
+
+/// Deserialize {"field": field} as { field ...} and serialize in reverse
+mod enum_field_as_inner {
```

Comment:
Im not sure about this name.

Also, I think i can use this for eventsub and pubsub deser instead of macro trickery, will investigate